docs/concepts/customizing-create-api.md

```diff
@@ -30,7 +29,7 @@ const customCreateApi = buildCreateApi(coreModule(), reactHooksModule({ useDispa
 
 ## Creating your own module
 
-If you want to create your own module, best take a lookt at [the react-hooks module](https://github.com/rtk-incubator/rtk-query/blob/next/src/react-hooks/module.ts) as what such an implementation would look like.
+If you want to create your own module, you should review [the react-hooks module](https://github.com/rtk-incubator/rtk-query/blob/next/src/react-hooks/module.ts) to see what an implementation would look like.
```

Comment:
:hand: should the link to go `main` instead of `next` ?

Reply:
Hmm, good question. Yeah, for documentation purposes `main` is probably better.